fission-rbac.yaml

```diff
@@ -0,0 +1,180 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: fission-svc
+  namespace: fission
+  labels:
+    kubernetes.io/cluster-service: "true"
```

Comment:
What's this label for? As I understand it, the cluster-service label is used for cluster addons... but i can't find some definitive documentation either way.